presto-in-memory/src/main/java/com/facebook/presto/plugin/inmemory/InMemoryMetadata.java

```diff
@@ -116,8 +117,9 @@ public ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTable
     @Override
     public void dropTable(ConnectorSession session, ConnectorTableHandle tableHandle)
     {
-        InMemoryTableHandle inMemoryTableHandle = checkType(tableHandle, InMemoryTableHandle.class, "tableHandle");
-        tables.remove(inMemoryTableHandle.getTableName());
+        // TODO: implement this method. Problem is that data are distributed across the cluster and this method
+        // is only being invoked on the coordinator
+        throw new UnsupportedOperationException("DROP TABLE is not supported");
```

Comment:
Per @maciejgrzybek, we should probably throw `PrestoException` with `NOT_SUPPORTED` `ErrorCode` instead of `java.lang.UnsupportedOperationException`.